events/templates/events/join_event.html

```diff
@@ -26,6 +26,8 @@ <h2 class="card-title">Registration for "{{ event.name }}"</h2>
             {% endfor %}
         {% endif %}
         <form method="post" novalidate>
+          <!-- If the user is the host of this event, they won't be able to join it. -->
```

Comment:
Imo I should probably change this in a separate pull request as it's not revelant to this. What I would do here is use the UserPassesTest mixin as well. I reckon just delete this from this branch for now.